lib/buildenvsetup/base.ts

```diff
@@ -84,7 +84,7 @@ export class BuildEnvSetupBase {
         } else if (this.compilerTypeOrGCC === 'clang') {
             const binpath = path.dirname(this.compiler.exe);
             const llc = path.join(binpath, 'llc');
-            if (fs.existsSync(llc)) {
+            if (await fs.exists(llc)) {
```

Comment:
fs.exists is officially deprecated, you can use await utils.fileexists though

Reply:
ta!